packages/dd-trace/src/plugins/util/web.js

```diff
@@ -361,6 +364,32 @@ const web = {
     const res = context.res
 
     scope.bind(res, context.span)
+  },
+  obfuscateQs (obfuscator, url) {
+    if (obfuscator === false) return url
+
+    const i = url.indexOf('?')
```

Comment:
Why not use `URL` to extract the query string for us and reconstruct the string?

Reply:
Just to simplify the code really. If you think the current implementation is simpler that's fine.